### DISCUSSION TITLE
全屏时移动鼠标不显示标题

### DISCUSSION THREAD

**Discussion by windwhim** [the1812/Bilibili-Evolved#3839] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/55055205/203494741-08697724-801e-4b7b-8fcc-74db400926f8.png)

**Comment by Ethkuil** [comment]
未能复现。

能确定是`Bilibili-Evolved`的问题吗（比如只要关了这个脚本问题就没了）？若是的话，装了哪些组件？

**Reply by the1812** [reply]
我猜是一顿猛装，装了个隐藏视频标题层还不知道（

**Reply by windwhim** [reply]
> 我猜是一顿猛装，装了个隐藏视频标题层还不知道（

那你可以仔细说说吗

**Reply by windwhim** [reply]
> 我猜是一顿猛装，装了个隐藏视频标题层还不知道（

谢谢，已经解决，是安装了隐藏视频标题层

**Reply by windwhim** [reply]
> 未能复现。
> 
> 能确定是`Bilibili-Evolved`的问题吗（比如只要关了这个脚本问题就没了）？若是的话，装了哪些组件？

不好意思，已经解决，是安装了隐藏视频标题层，建议可以将该组件添加**视频**标签。以为是在**视频**内，所以很久没有找到
